internal/action/run.go

```diff
@@ -153,6 +153,12 @@ func (s *service) watchFolder(action domain.Action, release domain.Release) erro
 	_, tmpFileName := path.Split(release.TorrentTmpFile)
 	fullFileName := path.Join(watchFolderArgs, tmpFileName+".torrent")
 
+	// Create folder
+	err := os.MkdirAll(watchFolderArgs, os.ModePerm);
```

Comment:
You need to change it to `err =` to not reassign the err.

Reply:
Can i add my fix to this pull request or do we need a new one?

Reply:
why is it ok to use := in line 147 and also again in 163?

Reply:
You can make a commit and push to your develop branch it this pr will update.

As to why - it's already been declared, and down on 163 we create two new variables so that's different. Not a great answer but that's how it is.